ui/plugin-system/src/components/PluginEditor/plugin-editor-api.ts

```diff
@@ -45,16 +48,17 @@ export type UsePluginEditorProps = Pick<PluginEditorProps, 'pluginType' | 'value
  * kind back.
  */
 export function usePluginEditor(props: UsePluginEditorProps) {
-  const { pluginType, value } = props;
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  const { pluginType, value, onHideQueryEditorChange = () => {} } = props; // setting onHideQueryEditorChange to empty functio here because useEvent requires a function
```

Comment:
```suggestion
  const { pluginType, value, onHideQueryEditorChange = () => {} } = props; // setting onHideQueryEditorChange to empty function here because useEvent requires a function
```

typo